package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bcrs-shared-components",
-  "version": "3.0.5",
+  "version": "3.0.6",
```

Comment:
This isn't needed because this file is for Storybook, and no content in this file has changed, and the components are separately versioned.

However, this doesn't hurt and I'll just merge it in anyways since everything else looks good.

Reply:
I have not published this PR (or any others recently). Let me know if you need them published asap so they can be imported in a Vue3 project.

Reply:
Good to know, thanks Sev! And no rush on publishing any of these components just yet, we still have a few more upgrades to complete.